src/content/overview/observability/logging/architecture/index.md

```diff
@@ -35,10 +38,21 @@ In this diagram, you can see that we run the following tools in each management
 
 - `Grafana Loki` that is accessible through our managed Grafana instance.
 - `multi-tenant-proxy`, a proxy component used to handle multi-tenancy for Loki.
-- A couple of logging agents (`Grafana Promtail` and `Grafana Agent`) that run on the management cluster and your workload clusters alike. We currently need two different tools for different purposes.
-    - Promtail is used to retrieve the container and kubernetes audit logs
+- A couple of scraping agents run on the management cluster and your workload clusters. There are different tools for different purposes:
+    - Promtail is used in older Giant Swarm releases to retrieve the container and kubernetes audit logs.
+    - Alloy is used in newer Giant Swarm releases to retrieve the container and kubernetes audit logs.
     - Grafana Agent is used to retrieve the kubernetes events.
 
+### Release compatibility
+
+Release|Alloy|Promtail|Grafana Agent|
```

Comment:
You need to add Promtail in the `.vale/config/accept.txt` words

Reply:
added